mdsa/tests/test_svd.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+import unittest
+
+from skbio.util import get_data_path
+
+from mdsa.algorithms.svd import Svd
+
+
+class TestSVD(unittest.TestCase):
```

Comment:
For consistency, if the class is Svd this should be TestSvd but I think changing, as in previous comment, to SVD, will be the best option.